fleetspeak/src/server/https/streaming_message_server.go

```diff
@@ -120,8 +120,12 @@ func (s streamingMessageServer) ServeHTTP(res http.ResponseWriter, req *http.Req
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
-	info, moreMsgs, err := s.initialPoll(ctx, addrFromString(req.RemoteAddr), cert.PublicKey, fullRes, body)
+	addr := addrFromString(req.RemoteAddr)
+	info, moreMsgs, err := s.initialPoll(ctx, addr, cert.PublicKey, fullRes, body)
 	if err != nil || info == nil {
+		if err != nil {
+			log.Warningf("Initial poll failed for %v: %v", addr, err)
```

Comment:
FYI, the design intent is for the makeError function in initialPoll to be called whenever there is an error. It both logs the error, and set the http error code.

Reply:
Ack. Refactored the code here to log the warnings in makeError only.